lib/schedule/data.ex

```diff
@@ -249,10 +252,14 @@ defmodule Schedule.Data do
   end
 
   @spec stations(t()) :: [Stop.t()]
-  def stations(%__MODULE__{stops: stops}) do
-    stops
-    |> Map.values()
-    |> Enum.filter(&Stop.is_station?/1)
```

Comment:
**question (non-blocking):** what's the reason for changing the call to `Stop.is_station?/1` to instead do the same check but without the function call?

Reply:
Ah I forgot about `is_station?` while making this change, will update!